src/runner.js

```diff
@@ -4,13 +4,28 @@ const parser = require('./parser/parser.js');
 const results = require('./results.js');
 const DefaultVariableStorage = require('./default-variable-storage.js');
 const nodeTypes = require('./parser/nodes.js').types;
+const MessageFormat = require('messageformat');
 
 class Runner {
-  constructor() {
+  /**
+   * Creates an instance of Runner.
+   * @param {object} [options] Options for the runner
+   * @param {object} options.formaterOptions Options for messageFormat formater
+   * @param {string} options.formaterOptions.language Options for the language of the formater
+   * @memberof Runner
+   */
+  constructor(options) {
     this.yarnNodes = {};
     this.variables = new DefaultVariableStorage();
     this.functions = {};
     this.visited = {}; // Which nodes have been visited
+    this.options = {
```

Comment:
to avoid deconstructing, here you can say

```
this.options = options;
this.options.language = this.options.language || "end";
```
then later on use `this.options.language` instead of `this.options.formaterOptions.language`

Reply:
yeah